### DISCUSSION TITLE
gy/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#935] [Announcements]
# gy/

关于bot 这个bot的建设的主要原因在于mc群聊的白名单服务，在2021年年末建立，逐渐完善框架，现在已经成为一个相对功能精简的多功能bot了，感谢很多大佬，同学，朋友的技术支持。 特别鸣谢 服务器支持 ste0807 联系方式 QQ: 3280577757 捐赠 Bot和Mc服务器的运行需要大量成本，如果您愿意支持，请捐赠。 image

https://docs.toomc.top/gy/

**Comment by YRRLYB** [comment]
必须支持😉
